pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -55,6 +57,7 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     django-cors-headers
     django-crispy-forms
     django-crispy-bootstrap4
+    django-debug-toolbar
```

Comment:
Still opposed to this, set `DEBUG=0` instead.

Reply:
If you apply the following diff, the application crashes with the same error as originally:

```patch
diff --git a/pkgs/applications/misc/tandoor-recipes/default.nix b/pkgs/applications/misc/tandoor-recipes/default.nix
index 66c85241dac1..39d28bb4aeb9 100644
--- a/pkgs/applications/misc/tandoor-recipes/default.nix
+++ b/pkgs/applications/misc/tandoor-recipes/default.nix
@@ -57,7 +57,6 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     django-cors-headers
     django-crispy-forms
     django-crispy-bootstrap4
-    django-debug-toolbar
     django-hcaptcha
     django-js-reverse
     django-oauth-toolkit
@@ -158,7 +157,7 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     tests = {
       inherit (nixosTests) tandoor-recipes;
       runTandoorRecipes = runCommand "run-${pname}" {} ''
-        ${tandoor-recipes}/bin/tandoor-recipes > $out
+        DEBUG=0 ${tandoor-recipes}/bin/tandoor-recipes > $out
       '';
       runManagePy = runCommand "run-${pname}" {} ''
         ${tandoor-recipes}/lib/tandoor-recipes/manage.py > $out
```

Reply:
Sorry, `DEBUG_TOOLBAR=0`.

Reply:
Same result:

```console
❯ git diff
diff --git a/pkgs/applications/misc/tandoor-recipes/default.nix b/pkgs/applications/misc/tandoor-recipes/default.nix
index 02e1e9fec4cb..ff2bf10baee5 100644
--- a/pkgs/applications/misc/tandoor-recipes/default.nix
+++ b/pkgs/applications/misc/tandoor-recipes/default.nix
@@ -56,7 +56,6 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     django-cors-headers
     django-crispy-forms
     django-crispy-bootstrap4
-    django-debug-toolbar
     django-hcaptcha
     django-js-reverse
     django-oauth-toolkit
@@ -163,7 +162,7 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     tests = {
       inherit (nixosTests) tandoor-recipes;
       runTandoorRecipes = runCommand "run-${pname}" {} ''
-        ${tandoor-recipes}/bin/tandoor-recipes > $out
+        DEBUG_TOOLBAR=0 ${tandoor-recipes}/bin/tandoor-recipes > $out
       '';
       runManagePy = runCommand "run-${pname}" {} ''
         ${tandoor-recipes}/lib/tandoor-recipes/manage.py > $out
❯ nix-build -A tandoor-recipes.passthru.tests
[…]
       > ModuleNotFoundError: No module named 'debug_toolbar'
```